Sources/PackageModel/Toolchain.swift

```diff
@@ -88,6 +88,15 @@ extension Toolchain {
         }
     }
 
+    public var toolchainDir: AbsolutePath {
+        get throws {
+            try resolveSymlinks(swiftCompilerPath)
+                .parentDirectory // bin
+                .parentDirectory // usr
+                .parentDirectory // <toolchain>
```

Comment:
This probably breaks for some platforms like FreeBSD where there's an expectation of using `/usr/local` but will handle that in a future change.